content/posts/efficient-navigation-in-vim.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Efficient Navigation in Vim
+date: 2020-07-07
+---
+
+When editing a file, it's quite crucial that one can navigate their cursor around rather quickly. [Vim](https://www.vim.org/) and [NeoVim](https://neovim.io/) allow for many different ways of doing so which, depending on the situation, can be more or less efficient and useful. This article examines the different ways of moving the cursor and compares them. Here, the term of "efficiency" refers to navigating the cursor with as little time and effort (i.e. the number of keystrokes) as possible.
+
+## Using the Mouse
+
+**TL;DR: Not recommended**
+
+Using the mouse pointer to navigate Vim may seem like an obvious choice for users coming from a GUI editor like [Visual Studio Code](https://code.visualstudio.com/) or similar. However, in Vim, the goal is to keep one's fingers on the "home row" of the keyboard (the row where the `F` and the `J` keys reside) and not having to move them greatly towards the touchpad or even the mouse. This should reduce the strain on one's hands and wrists and make editing more efficient. Therefore, navigating with the mouse should be highly discouraged in most situations and is even disabled by default.
+
+It can be helpful to enable it for users transitioning from one of the aforementioned editors and would like to have a smooth transition by allowing themselves to use the mouse initially and switching it back off again later in their learning journey. To turn on mouse navigation, add the following to the configuration file (either `~/.vimrc` or `~/.config/nvim/init.vim`):
+
+```vim
+" Temporarily enable mouse support
+set mouse=a
+```
+
+This will allow to point and click to move the cursor and to scroll through the current buffer.
```

Comment:
`This will allow you`: again nothing wrong with addressing the reader directly in my opinion